app/src/main/java/com/quran/labs/androidquran/ui/fragment/TabletFragment.java

```diff
@@ -319,16 +320,18 @@ protected void onPostExecute(RectF[] rect) {
           if (mRightImageView != null && mLeftImageView != null) {
             mRightImageView.setPageBounds(rect[0]);
             mLeftImageView.setPageBounds(rect[1]);
-            Context context = getContext();
-            if (mOverlayText && context != null) {
```

Comment:
good call, i agree. in `QuranPageFragment`, we're checking `isAdded()` and that seems to be working fine (no crash there), so am good with wrapping the entire block with an `isAdded()` check.